### ISSUE TITLE
About how to understand the meaning of this comparison in this paper？

### ISSUE THREAD

**Issue by Ruye-aa** [zhulf0804/NgeNet#3]
Thanks for your outstanding work!
I have a question about the comparison with PREDATOR as mentioned in your paper.

`However, different from PREDATOR (Huang et al., 2021) that focuses on the point sampling(i.e overlap and saliency scores), NgeNet pay more attention to the encoding of point features.`

As far as i' am concerned that the overlap score is about location information ,but the saliency scores in PREDATOR have the same meaning as the point features you mentioned. So i don't know whether there is anther meaning.
Best regards.
![image](https://user-images.githubusercontent.com/76813112/166216083-98ff6cd4-767f-4d28-8061-4f08fea34153.png)

**Comment by zhulf0804** [comment]
Thank you for your interest.

In my opinion, point features are used for matching to get correspondence in point cloud registration, while point attributes(properties) are used to select important points(such as overlap points, keypoints, salient points) for better registration. They are two factors for point cloud registration.

Here, we regard overlap and saliency scores  as point attributes that are effective for registration (see PREDATOR). We also learn overlap and saliency scores in our paper which is inspired by PREDATOR, while we focus on feature learning with the proposed module: ms + voting and GGE module.

Best regards.

**Comment by Ruye-aa** [comment]
So my understanding is to refine the attributes of points through better feature learning to perform point cloud registration.   Is that correct?
Thank you for your patient reply.

**Comment by zhulf0804** [comment]
Maybe not.

We didn't optimize to refine the point attributes in our work and adopted the off-of-the-shelf algorithm to obtain point attributes.

What we did is to learn a better feature descriptor through the proposed modules. Umm, as we know, learning-based features are generally better than hand-crafted features without considering data domains. Also, different learning-based features(such as FCGF, D3Feat) show different performances in point cloud registration. 

We conducted features comparison with PREDATOR directly, for we both used the same point attributes for point cloud registration. We intend to demonstrate that features improve the registration performance, whether considering the point attributes. As shown in Table 3(in the paper), whether random sampling or sampling  in prob. (based on overlap and saliency scores), NgeNet achieves higher recall than PREDATOR.  

Best regards.
